docs/docsite/rst/dev_guide/developing_modules_best_practices.rst

```diff
@@ -31,8 +31,8 @@ Designing module interfaces
 General guidelines & tips
 =========================
 
-* Each module should be self-contained in one file, so it can be auto-transferred by Ansible.
-* Module name MUST use underscores instead of hyphens or spaces as a word separator. Using hyphens and spaces will prevent Ansible from importing your module.
+* Each module should be self-contained in one file, so it can be auto-transferred by ansible-base.
+* Module name MUST use underscores instead of hyphens or spaces as a word separator. Using hyphens and spaces will prevent ansible-base from importing your module.
```

Comment:
```suggestion
* Module name MUST use underscores instead of hyphens or spaces as a word separator. Using hyphens and spaces will prevent ``ansible-base`` from importing your module.
```